agent-tooling/src/test/groovy/io/opentelemetry/auto/test/ConfigTest.groovy

```diff
@@ -0,0 +1,56 @@
+package io.opentelemetry.auto.test
+
+import io.opentelemetry.auto.config.AgentConfig
+import io.opentelemetry.auto.util.test.AgentSpecification
+
+class ConfigTest extends AgentSpecification {
```

Comment:
check out `io.opentelemetry.auto.api.ConfigTest` in case there is overlap with this